lib/active_merchant/billing/gateways/blue_snap.rb

```diff
@@ -491,13 +501,13 @@ def commit(action, verb = :post, payment_method_details = PaymentMethodDetails.n
           resource_url = "#{payment_method_details.resource_url}/#{options[:authorization]}/refund?cancelsubscriptions=false"
           resource_url += "&amount=#{options[:money]}" if options[:money].present?
           payment_method_details = OpenStruct.new(resource_url: resource_url)
-        else
```

Comment:
don't we need `else` anymore?

Reply:
now We have `elsif action != :get_payment_fields_token` so it works the same for all actions except `get_payment_fields_token`